### PR TITLE
Introduce session module, create Common plugin

### PR DESCRIPTION
**Pull request by ennru**
## Purpose

* Introduce a separate module as the new home for the Cassandra session
* Use a `Common` plugin for shared settings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# THIS IS WORK IN PROGRESS for version 1.0 of
```
Comment (2m) [inline README.md]: This is not needed anymore, as the repo is not being cloned.

File: version.sbt [hunk 2]
```diff
@@ -1 +1 @@
-version in ThisBuild := "0.99-SNAPSHOT"
+version in ThisBuild := "1.0-SNAPSHOT"
```
Comment (2m) [inline version.sbt]: Lets migrate to dynver instead?

**Comment by 2m** [comment]
Lets merge after #480 and #481 are merged. Because those PRs are for 0.9x and 1.x and this PR is for 1.x only.

File: README.md (hunk 1 above)
Comment (ennru) [reply README.md]: Let's clean that up together with the Paradox doc PR.

File: version.sbt (hunk 2 above)
Comment (ennru) [reply version.sbt]: Added a separate issue.

**Comment by ennru** [comment]
I added these changes to #414 instead.

**Pull request closed without merging**